src/main/java/uk/gov/hmcts/darts/authorisation/exception/AuthorisationError.java

```diff
@@ -65,6 +65,11 @@ public enum AuthorisationError implements DartsApiError {
         AuthorisationErrorCode.BAD_REQUEST_ANNOTATION_ID.getValue(),
         HttpStatus.BAD_REQUEST,
         AuthorisationTitleErrors.BAD_REQUEST_ANNOTATION_ID.getValue()
+    ),
+    USER_NOT_AUTHORISED_FOR_PAYLOAD_ENDPOINT(
+        AuthorisationErrorCode.USER_NOT_AUTHORISED_FOR_PAYLOAD_ENDPOINT.getValue(),
```

Comment:
maybe it's just me but I find the naming of this new error code to be not intuitive. What does "payload endpoint" mean?

Reply:
yeah this is an authorisation failure based on the payload content used. Maybe USER_NOT_AUTHORISED_TO_USE_PAYLOAD_CONTENT

Reply:
that's better thanks :)